public/javascripts/showprojectscript.js

```diff
@@ -126,12 +150,7 @@ function initNeume(neume, project, owned) {
             header = $(".row");
         roundedCorners = $(".roundedCorners");
 
-        if (y >= 50) {
-            header.css({ position: "fixed", "top": "57px" });
-
-        } else {
-            header.css({ position: "fixed", "top": "165px" });
-        }
+        header.css({ position: "fixed", "top": "120px" });
```

Comment:
This should probably be defined as a class that is added or removed to elements.